lms/djangoapps/instructor/access.py

```diff
@@ -75,6 +79,8 @@ def _change_access(course, user, level, action):
         raise ValueError("unrecognized level '{}'".format(level))
 
     if action == 'allow':
+        if level == 'ccx_coach':
+            before, after, enrollment_obj = enroll_email(course_id=course.id, student_email=user.email, auto_enroll=True)
```

Comment:
I see. 

I'd like to e-mail the coach. You should probably mention that in the cover letter as well. 

- Peter

> On Sep 30, 2015, at 11:59 AM, Jamie Folsom <notifications@github.com> wrote:
> 
> In lms/djangoapps/instructor/access.py <https://github.com/mitocw/edx-platform/pull/89#discussion_r40812919>:
> 
> > @@ -75,6 +79,8 @@ def _change_access(course, user, level, action):
> >          raise ValueError("unrecognized level '{}'".format(level))
> >  
> >      if action == 'allow':
> > +        if level == 'ccx_coach':
> > +            before, after, enrollment_obj = enroll_email(course_id=course.id, student_email=user.email, auto_enroll=True)
> Here's the call to enroll_email. And the method signature for enroll email is:
> 
> def enroll_email(course_id, student_email, auto_enroll=False, email_students=False, email_params=None, language=None):
> 
> Quoting the comments on that method, `email_students` determines if student should be notified of action by email.
> 
> So by not specifying that parameter, the default is used, and the new coach is not emailed to notify them they've been enrolled.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mitocw/edx-platform/pull/89/files#r40812919>.
>